ci/checks/copyright.py

```diff
@@ -197,7 +197,12 @@ def checkCopyright_main():
                                  "Can be specified multiple times."))
 
     (args, dirs) = argparser.parse_known_args()
-    ExemptFiles = [re.compile(pathName) for pathName in args.exclude]
+    try:
+        ExemptFiles = [re.compile(pathName) for pathName in args.exclude]
+    except re.error as reException:
+        print("Regular expression error:")
+        print(reException)
+        retval = 1
```

Comment:
Shouldnt this exit instead of continuing on?

Reply:
I made the change to return directly instead of continuing